processing/src/test/java/org/apache/carbondata/lcm/locks/LocalFileLockTest.java

```diff
@@ -68,4 +80,18 @@
     Assert.assertTrue(localLock2.unlock());
   }
 
+  @Test public void testConfigurablePathForLock() throws Exception {
+    Field f = secretClass.getDeclaredField("lockPath");
+    f.setAccessible(true);
+    f.set(secretClass, rootPath + "/target/");
+    AbsoluteTableIdentifier absoluteTableIdentifier = AbsoluteTableIdentifier
+        .from(CarbonProperties.getInstance().getProperty("carbon.storelocation"), "databaseName",
+            "tableName", "1");
```

Comment:
reset the lockpath in testcase finally clause

Reply:
done